Functions.Templates/Templates/IoTHubTrigger-CSharp/function.json

```diff
@@ -0,0 +1,13 @@
+{
+  "disabled": false,
+  "bindings": [
+    {
+      "type": "eventHubTrigger",
+      "name": "myIoTHubMessage",
+      "direction": "in",
+      "path": "samples-workitems",
```

Comment:
Path property is now changed to eventhub name 
https://github.com/Azure/azure-functions-templates/blob/dev/Functions.Templates/Bindings/bindings.json#L847